cinema/serializers.py

```diff
@@ -24,15 +24,13 @@ class Meta:
             "full_name"
         )
 
-    @staticmethod
     def get_full_name(obj):
         return f"{obj.first_name} {obj.last_name}"
 
```

Comment:
each method related to SerializerMethodField must have self parameter. fix for all methods